UnityProject/Assets/Scripts/Objects/Botany/SeedExtractor.cs

```diff
@@ -118,8 +118,23 @@ private void UpdateMe()
 		/// <param name="seedPacket">Seed packet to spawn</param>
 		public void DispenseSeedPacket(SeedPacket seedPacket)
 		{
-			// Spawn packet
 			Vector3 spawnPos = gameObject.RegisterTile().WorldPositionServer;
+			bool willSpawnFromInventory = false;
+			//spawn packet if added directly into inventory by player
+			foreach (ItemSlot slot in storage.GetItemSlots())
+			{
+				if (slot.IsEmpty) continue;
+				if (!slot.ItemAttributes.HasTrait(CommonTraits.Instance.Seeds)) continue;
+				if (slot.ItemObject == seedPacket.gameObject)
+				{
+					Inventory.ServerDrop(slot, spawnPos);
+					willSpawnFromInventory = true;
+					break;
+				}
+			}
+			if(willSpawnFromInventory) return;
```

Comment:
Sorry, what is this trying to do?

Reply:
Fixes the issue where seed packets that are put back into the extractor no longer become pickupable once dispensed again

Reply:
Humm, ok. Probably a better way to do it, think you can find out what slot an item is in from itself, you dont need to go looking for it. Might be on `Pickupable` component

Reply:
Ah alright, sounds much better. will try doing this.

Reply:
I have no idea if this works because Unity is being annoying but logically it should, someone please test the latest commit.

Reply:
tested and works just fine.